src/vaadin-pro-grid-edit-column.html

```diff
@@ -0,0 +1,190 @@
+<!--
+@license
+Copyright (c) 2018 Vaadin Ltd.
+This program is available under Commercial Vaadin Add-On License 3.0 (CVALv3).
+See <a href="https://vaadin.com/license/cval-3">the website</a> for the complete license.
+-->
+
+<link rel="import" href="../../polymer/lib/elements/dom-if.html">
+<link rel="import" href="../../vaadin-grid/vaadin-grid-column.html">
+<link rel="import" href="../../vaadin-text-field/vaadin-text-field.html">
+
+<dom-module id="vaadin-pro-grid-edit-column">
+  <template>
+    <template id="editStateTemplate">
+      <vaadin-text-field theme="small"></vaadin-text-field>
+    </template>
+  </template>
+
+  <script>
+  (function() {
+    /**
+     * `<vaadin-pro-grid-edit-column>` is a helper element for the `<vaadin-pro-grid>`
+     * that provides default inline editing for the items.
+     *
+     * #### Example:
+     * ```html
+     * <vaadin-pro-grid items="[[items]]">
+     *  <vaadin-pro-grid-edit-column path="name.first"></vaadin-pro-grid-edit-column>
+     *
+     *  <vaadin-grid-column>
+     *    ...
+     * ```
+     *
+     * @memberof Vaadin
+     * @extends Vaadin.GridColumnElement
+     */
+    class ProGridEditColumnElement extends Vaadin.GridColumnElement {
+
+      static get is() {
+        return 'vaadin-pro-grid-edit-column';
+      }
+
+      static get properties() {
+        return {
+          /**
+           * JS Path of the property in the item used for the editable content.
+           */
+          path: String
+        };
+      }
+
+      ready() {
+        super.ready();
+
+        this.__editStateTemplate = this._prepareTemplatizer(this.$.editStateTemplate);
+        // needed to override the dataHost correctly in case internal template is used.
+        this.__editStateTemplate.templatizer.dataHost = this;
+        this.__editStateRenderer = this.constructor._getEditorRenderer();
```

Comment:
As you scroll the grid while the cell is in edit mode, the same editor component will appear on an arbitrary row with the content of the cell being actually edited

Reply:
Discussed in person and it is OK for now to exit edit state when scrolled out of the viewport.

Reply:
Done.